### PR TITLE
New package: grip-4.5.2

### PR DESCRIPTION
**Pull request by LinArcX**
https://github.com/void-linux/void-packages/issues/20002

### CODE REVIEW

**Comment by Johnnynator** [comment]
Pls package this as python3 packages. We do not accept any new Py2 packages.

**Comment by LinArcX** [comment]
@Johnnynator Replacing python-module with python3-module and removing is enough?

**Comment by Johnnynator** [comment]
Rename the packages to `python3-name` and all deps need to be changed to their python3 name.

**Comment by LinArcX** [comment]
Done:
https://github.com/void-linux/void-packages/pull/20076

**Comment by Chocimier** [comment]
python-grip is mainly commandline tool rather than module, so should be named grip.

File: srcpkgs/grip/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+depends="python3-docopt python3-Flask python3-Markdown python3-path-and-address python3-Pygments python3-requests"
+short_desc="Preview GitHub Markdown files like Readme"
+maintainer="linarcx <linarcx@riseup.net>"
+license="MIT"
+homepage="https://github.com/joeyespo/grip"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=bdf8949f33470e9ef9e3f09596b72cda968116ff32f0280baabe837c2ad1b29b
+
+post_install() {
+	vlicense LICENSE LICENSE
```
[the remark this answers is not in the document]
Comment (LinArcX) [reply srcpkgs/grip/template]: Done!

File: srcpkgs/grip/template [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'grip'
+pkgname=grip
+version=4.5.2
+revision=1
+archs=noarch
+wrksrc="grip-${version}"
```
[the remark this answers is not in the document]
Comment (LinArcX) [reply srcpkgs/grip/template]: Done!

**Pull request merged**